webapp/src/components/Registration/Login/login.js

```diff
@@ -20,8 +19,16 @@ class Login extends Component {
 
   handleSubmit = (evt) => {
     evt.preventDefault();
-    this.props.onAuthenticate(this.state.email, this.state.password);
-    return this.setState({ error: '' });
+    const { email, password } = this.state;
+    fetch(`http://localhost:8888/registration?email=${email}&password=${password}`, {
```

Comment:
Any logic to doing this as a query string as opposed to body? I haven't seen anything that obfuscates the plaintext password that would be sent along here in the URL string.

Reply:
I only did it because it was easy. Should I move it to the body?

Reply:
Yeah, I think it's generally best practice to move this stuff out of the URL. In the body you can still be snooped on if the traffic is decrypted, but the log files on the server, prying eyes, history, etc. won't be encoding a password.